### DISCUSSION TITLE
It doesn't work in some browsers...

### DISCUSSION THREAD

**Discussion by B3ni15** [prplwtf/fetch-nebula#99] [Support]
So, when I set up the nebula, it works perfectly for me, but not for a friend. It works for me on firefox and chrome, but nowhere on his. You get this message:

![image](https://github.com/user-attachments/assets/4994aa85-3cbd-4a49-96b3-9d1626db5bb4)

I did what you said and it didn't help, I updated the panel, it wasn't good either, I updated the list of browsers and it didn't work either. **What could be the problem then?**

_If anyone has any ideas, please write!_

**Comment by prplwtf** [comment]
Could you verify that Pterodactyl works on the same browser without Nebula? Alongside that, could you let me know which browser they are using (and their user agent)?
